rulesets/nixpkgs/require-merge-queue.json

```diff
@@ -22,8 +22,8 @@
         "min_entries_to_merge": 1,
         "max_entries_to_merge": 5,
         "min_entries_to_merge_wait_minutes": 5,
```

Comment:
While we're talking about finessing the numbers, we should look at tweaking these numbers soon.

In particular, I think we should:
- Increase "max entries" (build & merge), so that we test & merge larger groups
- Increase "min entries," so that we don't start testing until we have a few PRs queued
- Consider decreasing "minutes to wait," so that we don't have to wait too long if the queue is really quiet for some reason (maybe 5m is ok already, though?)

Increasing the "max entries" is based on the assumption that failures will be rare. Therefore, we should optimistically attempt to test & merge larger groups, thereby speeding up the queue. Failures will delay more PRs, but the happy path will be faster.